lib/vsc/install/commontest.py

```diff
@@ -63,6 +64,11 @@
     except ImportError:
         pass
 
+# Prospector doesn't have support for 3.5 / 3.6
+# https://github.com/PyCQA/prospector/issues/233
+if sys.version_info >= (3, 5):
+    HAS_PROTECTOR = False
```

Comment:
i meant this (old, not your) typo: `HAS_PROTECTOR` -> `HAS_PROSPECTOR`

Reply:
haha, I didn't even notice that bit! In that variable names should be meaningful, and in that this is referring to prospector, I too think this should be corrected. I don't see the variable anywhere else in the vsc-install and did a grep for vsc-base as well, so as long as it's not imported in any of the lower lower vsc tools modules, it should be ok to change.